grr/core/grr_response_core/config/server.py

```diff
@@ -282,6 +282,15 @@
     "If the average network usage per client becomes "
     "greater than this limit, the hunt gets stopped.")
 
+# GRRafana HTTP Server settings.
+config_lib.DEFINE_string("GRRafana.server",
```

Comment:
I'd suggest using "bind" here as is in https://github.com/google/grr/blob/252af5d85f02121a75f8d83ebbee8e3d7e4f5fc9/grr/test/grr_response_test/test_data/grr_test.yaml#L238.